### PR TITLE
Permission handler bug fix

### PR DESCRIPTION
**Pull request by heitorcolangelo**
Issue [#102](https://github.com/concretesolutions/kappuccino/issues/102)

This is an attempt to fix the bug on `runtimePermission` method. Now the match is not made using the index 0 or 1, it's using the proper id of each button. It seems safer than using `int`s.
I had also added documentation to make it more clear the real purpose of that method.
Aside of that, I not so sure if this will fix once for all the bug, I would probably gonna need help testing this in different devices.
@alancamargo92 in which device where you testing when you got that error?

### CODE REVIEW

**Comment by alancamargo92** [comment]
Good stuff mate!
I tested it with my own device (LG G4), now I can test it again to make sure it's been fixed at least on my device.

**Comment by heitorcolangelo** [comment]
Cool, I'll test with an LG device as well to se if it fixes, because on emulator and on my Moto G5 I couldn't reproduce the error.
Do you mind in make a code review on this PR?

**Comment by alancamargo92** [comment]
Sure! Will do as soon as I can

File: sample/src/main/java/br/com/concretesolutions/kappuccino/sample/RuntimePermissionActivity.java [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
 import android.widget.Button;
 import android.widget.TextView;
 
-public class RuntimePermissionActivity extends AppCompatActivity implements ActivityCompat.OnRequestPermissionsResultCallback {
+public class RuntimePermissionActivity
+        extends AppCompatActivity
+        implements ActivityCompat.OnRequestPermissionsResultCallback {
 
     private static final int MY_PERMISSIONS_REQUEST_READ_CONTACTS = 123;
+    private static final String TAG = "TAG_ " + RuntimePermissionActivity.class.getSimpleName();
```
Comment (heitorcolangelo) [inline sample/src/main/java/br/com/concretesolutions/kappuccino/sample/RuntimePermissionActivity.java]: Unecessary space here after `TAG_` ... =/

File: kappuccino/src/main/kotlin/br/com/concretesolutions/kappuccino/custom/runtimePermission/runtimePermission.kt [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                if (allowPermissions.exists()) {
+                    allowPermissions.click()
+                } else {
+                    println("Could not find button with Id: $actionButtonId")
+                }
+
+            }
+
+        } catch (e: UiObjectNotFoundException) {
+            println("There is no permissions dialog to interaction with")
```
Comment (alancamargo92) [inline kappuccino/src/main/kotlin/br/com/concretesolutions/kappuccino/custom/runtimePermission/runtimePermission.kt]: A little typo here: ~interaction~ interact

**Comment by heitorcolangelo** [comment]
@alancamargo92 done!

**Pull request merged**